### ISSUE TITLE
Only 15 seconds to enter guest user nickname

### ISSUE THREAD

**Issue by fcharlaix-opendsi** [nextcloud/richdocuments#2596]
**Describe the bug**
On a public link to a document with edition right, you have only 15 seconds to enter a nickname, after this delay you are kick from Collabora.

**To Reproduce**
1. Create a new document
2. Share the document with a public link
3. Grant edition to the public link
4. On a navigator private session or different from the Nextcloud, open the public link
5. Wait 15 seconds on the nickname prompt

**Expected behavior**
Have plenty of time for any type of user to enter a nickname.
Configuration available to admin to set the nickname prompt timeout.
Possibility to retry if the prompt is closed.

**Screenshots**
The nickname prompt :
![image](https://user-images.githubusercontent.com/93119031/199685729-2b6c7ff7-a786-4ae6-8d91-d2bf98ad63ef.png)

The result after 15s wait :
![image](https://user-images.githubusercontent.com/93119031/199685961-ae9d988c-9549-49a5-8120-c2a08a3da949.png)

**Client details:**
 - OS: Ach Linux - Linux 6.0.6
 - Browser Vivaldi, FireFox
 - Version 5.5.2805.42, 106.0.3
 - Device: Desktop

## Server details

**Operating system:**
1. Debian 11 - Docker
2. Debian 10
3. Debian 11

**Web server:**
1. Nginx
2. Apache2
3. Apache2

**Database:**
1. PostgreSQL
2. MariaDB
3. MariaDB

**PHP version:**
1. 8.0
2. 7.4
3. 7.4

**Nextcloud version:**
1. 24.0.6
2. 23.0.0
3. 24.0.6

**Version of the richdocuments app:**
1. 6.3.1
2. 5.0.3
3. 6.3.1

**Version of Collabora Online:**
1. 22.05.7.3
2. 6.4.13
3. 22.05.7.3

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
No related logs found
```

#### Browser log
```
No error in the JavaScript console or the network logs
```

</details>

**Comment by Raudius** [comment]
Seems to be the timeout which checks if Collabora is taking too long to load. A workaround for now would be to increase the `timeout` app-value.

Perhaps we should not initiate loading the Collabora iframe until the nickname has been set

**Comment by juliushaertl** [comment]
We already emit a post message, so maybe we can use that to stop the timeout handling https://github.com/nextcloud/richdocuments/commit/e0edcbc463087a5edb337af8ac2c71f9829bd643

**Comment by aarushisoni** [comment]
Hi ! I want to contribute to this issue ... Kindly guide me how to proceed further

**Comment by joshtrichards** [comment]
Fixed in #3256
